src/components/lost-found/LostFoundNotificationBell.tsx

```
@@ -0,0 +1,332 @@
+                        {NOTIFICATION_TYPE_ICONS[notif.type]}
+                      </span>
+                      <div className="flex-1 min-w-0">
+                        <div className="flex items-center gap-2">
+                          <p className="text-xs font-bold text-black truncate">{notif.title}</p>
+                          {!notif.read && (
+                            <div className="h-2 w-2 shrink-0 rounded-full bg-peach" />
+                          )}
+                        </div>
+                        <p className="text-[10px] text-black/50 truncate mt-0.5">{notif.body}</p>
+                        <div className="flex items-center gap-2 mt-1">
+                          <span className="text-[9px] font-mono text-black/30">{timeAgo(notif.created_at)}</span>
```

ESLint

[error] Replace `{timeAgo(notif.created_at)}` with `⏎····························{timeAgo(notif.created_at)}⏎··························`

(prettier/prettier)